registration/tests/utils.py

```diff
@@ -35,6 +35,25 @@
 
 # Classes #####################################################################
 
+
+class ServerValidationComplete:
+    """
+    Waits for the Angular validation classes to be set on the element.
+    """
+
+    ng_validated_classes = ('ng-valid', 'ng-invalid')
+
+    def __init__(self, locator):
+        self.locator = locator
+
+    def __call__(self, driver):
+        element = driver.find_element(*self.locator)
+        classes = element.get_attribute('class').split(' ')
+        if 'ng-dirty' in classes and any(ng_class in classes for ng_class in self.ng_validated_classes):
```

Comment:
Could you please add some more explanation here or in the class docstring?
You have explained this workaround nicely in the comment, but it would be good to have it persistent in the code.

Reply:
:+1: